docs/application/dotnet/guides/app-management/overview.md

```diff
@@ -40,15 +32,6 @@ The application can communicate with other applications in various ways. You can
 
 You can use the following application data exchange features in your .NET applications:
 
--   [Message Port](message-port.md)
-
-    Applications communicate with each other using message ports. You can send and receive messages using the map data format and trusted or untrusted message port instances.
-
--   [Data Control](data-control.md)
```

Comment:
FYI : 
In this PR, there are several files that changed the location.

If a file changed the location, other files which have link to that file should be also checked and update the hyperlink, to prevent the 404 error.

In case of previous location for :
 - data-control.md
   applications/uiapplication/widget-app.md has link to this file so the hyperlink is broken.
- message-port.md
   applications/uiapplication/widget-app.md has link to this file so the hyperlink is broken.
   ```...sockets and [message ports](../app-management/message-port.md), ...```
- event.md
   guides/index.md has link to this file so the hyperlink is broken.
- app-controls.md
   guides/notification/attach-panel.md has link to this file so the hyperlink is broken.
   ```... using [application controls](../app-management/app-controls.md).```

Please update the location of the file in the hyperlink in these files also to fix the broken link.

Reply:
@mijin85cho 
How about this message: "Check Broken Link: Not found any broken link" ? This tool don't work properly?

Reply:
Fixed: links checked manually. Also I found some missing things in index.md file for other guides.